src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -4403,12 +4465,199 @@ subroutine cat_enkf_increments(                               &
           
        end do
 
+       ! ----------------------------------       
+       
+    case (11) select_update_type  ! 1d snow analysis (Toure et al. 2018 empirical gain); snow cover fraction obs
+       
+       if (logit) write (logunit, *) 'get 1d snow increments (Toure et al. 2018 empirical gain); snow cover fraction obs'
+       
+       ! ensure that max SWE increment parameter is less than WEMIN; larger increments make no sense because
+       ! at SWE=WEMIN, the tile is fully snow covered (asnow=1)
+
+       if (SCF_ANA_MAXINCRSWE>WEMIN)  call ldas_abort(LDAS_GENERIC_ERROR, Iam, 'must use SCF_ANA_MAXINCRSWE<=WEMIN')
+       
+       ! get target for snow layer thickness (as used in Catchment over land tiles)
+       
+       targetthick(1)        = CATCH_DZ1MAX
+       
+       targetthick(2:N_snow) = 1./(N_snow-1.)
```

Comment:
Calculation of targetthick(2:N_snow) here is potentially non-0-diff vs. the previously hardcoded DZMAX.  If testing of SCF DA runs returns non-0-diff results, change temporarily to `targetthick(2:N_snow)=0.5`